README.md

```diff
@@ -30,44 +27,47 @@ npm install -g @stoplight/spectral-cli
 yarn global add @stoplight/spectral-cli
 ```
 
-Find more [installation methods](https://meta.stoplight.io/docs/spectral/docs/getting-started/2-installation.md) in our documentation.
+Find more [installation methods](https://meta.stoplight.io/docs/spectral/ZG9jOjYyMDc0Mw-installation) in our documentation.
 
-**Create a Ruleset**
+**Create a local ruleset**
```

Comment:
@philsturgeon I think the changes are great and we can merge it as is, just a suggestion as I was reading the README updates, what do you think of breaking up Installation and Usage into two separate sections, and then having this heading and the next be numbered? So this section would be:

## Usage
...
**1. Create a Ruleset**
...
**2. Lint**
...

Just to make it clearer the user has to create a local ruleset.

Reply:
I've been trying to get these simple changes in for ages so I'd rather not add anything else to it at this point. 😆  I have other PRs to send for spectral docs.